docs/one-click-integrations/github-integration.md

```diff
@@ -115,12 +140,26 @@ The table below lists the data that the GitHub integration collects from your Gi
     <td>
         <p>Deployments:</p>
         <ul>
-            <li><code>deploy_id</code>: pull request ID</li>
+            <li><code>deploy_id</code>: pull request identifier</li>
+            <li><code>timestamp_created</code>: pull request merged date</li>
             <li><code>system</code>: repository name</li>
         </ul>
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
+<td>Pull requests</td>
+    <td>
+        <p>Incidents:</p>
+        <ul>
+            <li><code>incident_id</code>: pull request number</li>
```

Comment:
https://github.com/codacy/pulse-user-docs/pull/194/files#r1107053946

```suggestion
            <li><code>incident_id</code>: pull request identifier</li>
```

Reply:
Yeah, as I mentioned in the Bitbucket PR, GitHub actually has a unique PR identifier which we use for this purpose.